.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Creating PR in "react-native-jolocom" repo'
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Create pull request
+      uses: SBub/create-outside-pr@v1.0.0
+
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
```

Comment:
Every devs must have such key with all exposed rights to pass this job? can it be impersonated as well? for example with jolocom dev acc?

Reply:
the API_TOKEN_GITHUB is a secret of the `jolocom-sdk` the action reads it from there